ipn/store.go

```diff
@@ -14,14 +14,66 @@ import (
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
 	"sync"
 	"time"
 
 	"tailscale.com/atomicfile"
 	"tailscale.com/kube"
+	_ "tailscale.com/kube"
 	"tailscale.com/paths"
 )
 
+// NewStoreFunc returns a StateStore for the provided path.
```

Comment:
clarify that the path is the part after the colon?

Reply:
That's dependent on the implementation. Added comments in `RegisterStore` and `NewStore`

Reply:
These docs are all too indirect.

They need to be fleshed out to be more useful without chasing a few levels of indirection to find the content.